src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+const { getErrorWithStatus } = require('./getError');
+
+const getValidId = (id, errorMessage) => {
+  const numberId = Number(id);
+
+  if (isNaN(numberId)) {
+    throw getErrorWithStatus(400, errorMessage);
+  }
+
+  return numberId;
+};
+
+const getValidString = (value, fieldName, expectedType = '') => {
+  if (typeof value !== 'string') {
+    throw getErrorWithStatus(400, `Type of ${fieldName} must be string`);
```

Comment:
Consider adding a check for empty strings in `getValidString` if empty strings are not allowed. This can be done by adding `|| value.trim() === ''` to the condition.